docs/development/testing_guide.rst

```diff
@@ -41,8 +41,8 @@ multiple software components work together as intended.
 PlasmaPy's tests are set up using the pytest_ framework. The tests for
 a subpackage are located in its :file:`tests` subdirectory in files with
 names of the form :file:`test_*.py`. For example, tests for
-`plasmapy.formulary.parameters` are located at
-:file:`plasmapy/formulary/tests/test_parameters.py` relative to the top
+`plasmapy.formulary.speeds` are located at
+:file:`plasmapy/formulary/tests/test_speeds.py` relative to the top
```

Comment:
I think I chose `parameters` here because I figured I wouldn't even need to change it!  😹